openfl/native/native.py

```diff
@@ -215,7 +215,7 @@ def init(
     Args:
         workspace_template (str): The template that should be used as the
             basis for the experiment.  Defaults to 'default'.
-            Other options include are any of the template names
+            Other options include any of the template names
```

Comment:
I think we can omit this from this PR. Also because native API will be deprecated and removed eventually anyway